main.py

```diff
@@ -0,0 +1,54 @@
+from PIL import Image, ImageDraw, ImageFont
+import hashlib
+import os
+from mkdocs.structure.files import File
+
+
+def define_env(env):
+
+    @env.macro
+    def console(description, txt):
+        # Check if image exists already and has the same hash, if so, return the image
+        hash = description + txt
+        hash = hashlib.md5(hash.encode('utf-8')).hexdigest()
+        file_path = 'images/console_gen_' + hash + '.png'
+        p = os.path.dirname(env.page.url).split("/")
+        p = p[:-1]
+        relative_path = os.path.join(*p, file_path)
+        p = os.path.join("docs", *p, file_path)
+
+        documentation_image_path = '![' + description + \
+            '](' + file_path + " \"" + description + "\")"
```

Comment:
```suggestion
        documentation_image_path = '![' + description + '](' + file_path + " \"" + description + "\")"
```
For Python please use 120 characters per line.

Reply:
This is autopep8 default, not changing this one